modules/odc-creating-apiserversource.adoc

```diff
@@ -7,9 +7,16 @@
 
 .Procedure
 
-. Navigate to the **Add** page and select **Event Source**.
-. In the **Event Sources** page, select **ApiServerSource** in the **Type** section.
-. Configure the **ApiServerSource** settings:
+. In the **Developer** perspective, navigate to **+Add** → **Event Source**. The  **Event Sources** page in the **Developer Catalog** is displayed.
++
+If you have multiple providers, you can filter the available event sources based on the **Provider**.
```

Comment:
Add this as an optional step:
. Optional: If you have multiple providers for your event sources, select the required provider from the *Providers* list to filter the available event sources from the provider.

Reply:
Done.